src/com/galactanet/gametable/IGametablePlugin.java

```diff
@@ -0,0 +1,10 @@
+package com.galactanet.gametable;
+
+/**
+ * All plugins should implement this interface.
+ * When the gametable is ready for the plugin to start, it will call the initialise method with the gametable instance.
+ */
+public interface IGametablePlugin {
+    void initialise(GametableFrame gametable);
```

Comment:
Yeah, that makes sense, and shouldn't be too far from what is already there. Passing the frame though was mostly to reduce the amount of work needed at this stage, but totally agree that it should be more specific as it develops